starters/next12-routing-chakra-ui/__tests__/counter-example.test.tsx

```diff
@@ -0,0 +1,68 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import CounterExample from "../src/pages/counter-example/index";
+import "@testing-library/jest-dom";
```

Comment:
Is this needed to be imported in the test file? I thought the testEnvironment option in the jest config handled that?

Reply:
If I try to remove that import, then 4 tests will fail. 
<img width="918" alt="Screen Shot 2022-12-09 at 12 54 49 PM" src="https://user-images.githubusercontent.com/67210629/206794575-cfcb0e03-ca35-4e4f-a91a-5435083caca5.png">

<img width="922" alt="Screen Shot 2022-12-09 at 12 55 11 PM" src="https://user-images.githubusercontent.com/67210629/206794648-b197b21b-89f1-475a-8292-7556cdebf3e3.png">